src/diffusers/models/autoencoders/vae.py

```diff
@@ -926,6 +926,7 @@ def __init__(
         block_out_channels: Tuple[int, ...],
         upsampling_scaling_factor: int,
         act_fn: str,
+        upsample_fn: str,
```

Comment:
We need to have the default of `Upsample` here.

Reply:
Maybe this is not necessary. I checked the code, DecoderTiny will only be called when AutoencoderTiny is initialized, and upsample_fn is bound to pass parameters, consistent with act_fn. The default value of nn.Upsample is "nearest", which upsample_fn will pass. So the change will not affect the original TAESD.

Reply:
Okay.